[LTP] Correct passing LTP tests

dimakuv opened the pull request:
The previous commit 6f64f3a was introduced before Graphene fixed its IPC
subsystem (Graphene's commits between 48a01df and 89a25d1). This
resulted in the PASSED file updated incorrectly. This commit fixes some
incorrect changes in PASSED.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/oscarlab/graphene-tests/8)
<!-- Reviewable:end -->

mkow reviewed (approved):
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene-tests/8)* status: all files reviewed, all discussions resolved, not enough approvals from maintainers (1 more required)



<!-- Sent from Reviewable.io -->

Commit:
[LTP] Correct passing LTP tests

The previous commit 6f64f3a was introduced before Graphene fixed its IPC
subsystem (Graphene's commits between 48a01df and 89a25d1). This
resulted in the PASSED file updated incorrectly. This commit fixes some
incorrect changes in PASSED.

donporter reviewed (approved):
> Reviewed 1 of 1 files at r1.
> *[Reviewable](https://reviewable.io/reviews/oscarlab/graphene-tests/8)* status: :shipit: complete! all files reviewed, all discussions resolved



<!-- Sent from Reviewable.io -->
